custom/icds_reports/models/aggregate.py

```diff
@@ -779,6 +782,10 @@ class AggChildHealth(models.Model, AggregateMixin):
     zscore_grading_hfa_recorded_in_month = models.IntegerField(blank=True, null=True)
     zscore_grading_wfh_recorded_in_month = models.IntegerField(blank=True, null=True)
     lunch_count_21_days = models.IntegerField(blank=True, null=True)
+    live_birth_permanent_resident = models.IntegerField(blank=True, null=True)
+    live_birth_temp_resident = models.IntegerField(blank=True, null=True)
```

Comment:
I think these two columns are probably unnecessary. UCRs tend to have columns for every combination because they allow far more limited logic. What this records can be calculated from other columns in the report query with limited if any performance cost

Reply:
It looks like you are even already doing this

Reply:
could you please elaborate on this? I could have pulled these from agg_level=5 but the cost would go higher. It would be like loading a tabular report with the state selected and view by awc. such a tabular report takes minutes to load. which won't be great on a UI report?.